File: Good, good Python/step 06/6.01/6.1.03.py
'''

Подвиг 5. Вводятся данные в формате ключ=значение в одну строчку через пробел. 
Необходимо на их основе создать словарь, затем проверить, существуют ли в нем ключи со значениями: 'house', 'True' и '5' (все ключи - строки). 
Если все они существуют, то вывести на экран ДА, иначе - НЕТ.

Sample Input:

вологда=город house=дом True=1 5=отлично 9=божественно

Sample Output:

ДА

'''

lst_in = [i.split("=") for i in input().split()]
lst_in = [[i[0], i[-1]] for i in lst_in]
d = dict(lst_in)
if 'house' in d and 'True' in d and '5' in d:
    print('ДА')
else:
    print('НЕТ')

File: JavaScript. A3 Tasks/6  Задачи по темам/6.01/6.1.03.js
/*

Дана величина угла в градусах -- положительное натуральное число.

Переведите величину угла в радианы по формуле radians=degrees⋅2π360radians=degrees⋅3602π​

Sample Input 1:

180

Sample Output 1:

3.1416

Sample Input 2:

57

Sample Output 2:

0.9948

*/

let degrees = Number(prompt());
let radians = degrees * (2*Math.PI/360);
console.log(radians.toFixed(4));
